amida-tech/indaba-client
# Bug/INBA-730 Fix no feedback on password reset request

#### What does this PR do?
Fix the forgot password call handling to display feedback to the user on success

#### Related JIRA tickets:
[INBA-730](https://jira.amida-tech.com/browse/INBA-730)

#### How should this be manually tested?
1. Set a valid `GMAIL_PASSWORD` environment variable on greyscale so that emails can be sent
1. Enter a user's valid email in the login box and click forgot password
1. Click Reset Password
1. Check that after the call to greyscale, the front end routes back to /login and displays a success message

#### Background/Context

#### Screenshots (if appropriate):
